packages/shorebird_code_push_protocol/lib/src/models/release_platform.dart

```diff
@@ -1,12 +1,15 @@
 /// A platform to which a Shorebird release can be deployed.
 enum ReleasePlatform {
   // ignore: public_member_api_docs
-  android('Android'),
+  android('Android', 'aab'),
```

Comment:
I wonder if I need to extract this change to its own PR? It slipped my attention that this file is in a different package...